conf/nginx/templates/nginx.conf.web.template

```diff
@@ -89,6 +89,12 @@ http
         zmauth_admin;
     }
 
+    #  AUTH-417 access auth via consul sidecar upstream
```

Comment:
```suggestion
```
we can avoid this, also if possible move this below `tasks` upstream to keep things ordered :wink: